src/library/dlgtrackinfo.cpp

```diff
@@ -515,48 +555,92 @@ void DlgTrackInfo::clear() {
     m_pBeatsClone.clear();
 
     txtLocation->setText("");
+    txtBitrate->setText("");
```

Comment:
Why is this needed? Doesn't resetTrackRecord() reset all those fields implicitly? We should check that.